package.json

```diff
@@ -7,6 +7,7 @@
     "dev": "next dev -p 8080 ",
     "export": "next build && next export -o dist",
     "test": "jest",
+    "test:watch": "jest --watch",
```

Comment:
This is not strictly related to the `comments.test` fixes but I wanted a way to iterate on tests with a watcher.